proto/api.proto

```diff
@@ -903,11 +898,13 @@ message TradeSwapResponse{
 }
 
 enum Project {
-  P_UNKNOWN = 0;
-  P_JUPITER = 1;
-  P_RAYDIUM = 2;
-  P_SERUM = 3;
-  P_ALL = 4;
+  P_ALL = 0;
```

Comment:
wait, UNKNOWN should stay as the zero value here, so if the user doesn't include a value at all we shouldn't make assumptions about if ALL is the value they want